sdk/tests/conformance2/rendering/depth-stencil-feedback-loop.html

```diff
@@ -109,8 +109,15 @@
     tex1 = gl.createTexture();
     tex2 = gl.createTexture();
     wtu.fillTexture(gl, tex0, width, height, [0x0, 0xff, 0x0, 0xff], 0, gl.RGBA, gl.UNSIGNED_BYTE, gl.RGBA);
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
```

Comment:
These changes were already merged under an earlier pull request. Could you clean this up? Thanks.